### ISSUE TITLE
[CMake error]  can't build llvm tools.

### ISSUE THREAD

**Issue by miz2019** [avast/retdec#736]
```
diff --git a/deps/llvm/CMakeLists.txt b/deps/llvm/CMakeLists.txt
index 820f07aa8..1a7624e3a 100644
--- a/deps/llvm/CMakeLists.txt
+++ b/deps/llvm/CMakeLists.txt
@@ -67,7 +67,7 @@ if(LLVM_LOCAL_DIR)
                        -DLLVM_ENABLE_WARNINGS=NO

                        # Disable the generation of targets for tests (we don't need them).
-                       -DLLVM_INCLUDE_TOOLS=OFF
+            -DLLVM_INCLUDE_TOOLS=ON
                        -DLLVM_INCLUDE_UTILS=OFF
                        -DLLVM_INCLUDE_RUNTIMES=OFF
                        -DLLVM_INCLUDE_EXAMPLES=OFF
@@ -77,7 +77,7 @@ if(LLVM_LOCAL_DIR)
                        -DLLVM_INCLUDE_DOCS=OFF

                        # Disable build of unnecessary LLVM parts.
-                       -DLLVM_BUILD_TOOLS=OFF
+            -DLLVM_BUILD_TOOLS=ON
                        -DLLVM_BUILD_UTILS=OFF
                        -DLLVM_BUILD_RUNTIMES=OFF
                        -DLLVM_BUILD_RUNTIME=OFF
```
I changed the llvm cmake to generate some tools. but it failed. The line number may not be right, because i change some cmake to use locally download package for bad internet connection.
The error is below

```
[ 16%] Built target retdec-configtool
-- Targeting X86
CMake Error at cmake/modules/LLVM-Config.cmake:175 (message):
  Target X86 is not in the set of libraries.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:548 (llvm_map_components_to_libnames)
  cmake/modules/AddLLVM.cmake:626 (llvm_add_library)
  tools/lto/CMakeLists.txt:19 (add_llvm_library)
```

Thanks.

**Comment by s3rvac** [comment]
Hi! We use a [modified version of LLVM](https://github.com/avast/llvm) that (1) does not perform any optimizations that decrease the quality of RetDec's output and that (2) contains only the parts the we need (to speedup the build). We have disabled the build of LLVM tools as we do not need them + we are not even sure that our modified version of LLVM can build them. Based on your report, build of LLVM tools does not work. The issue that you have reported is similar to #701.

So, if anyone is interested in building the tools, they will have to modify [our LLVM fork](https://github.com/avast/llvm) to ensure that they can be built. However, it probably will not be us as we do not need them and we have disabled their build on purpose. We would definitely welcome a patch in the form of a pull request though.
